english/boston/boston.md

```diff
@@ -4,6 +4,8 @@
 
 ...or on the second thought, I'll put on a coat, stop being a sissy, and get to work!
 
+I'm very lucky to live in the Avalon Back Bay apartment complex, it is not only very close to Copley Square, but it is also inside a shopping mall!
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.